src/go/k8s/pkg/resources/certmanager/schemaregistry_api.go

```diff
@@ -20,6 +20,13 @@ const (
 
 // SchemaRegistryAPINodeCert returns the namespaced name for the SchemaRegistry API certificate used by nodes
 func (r *PkiReconciler) SchemaRegistryAPINodeCert() types.NamespacedName {
+	tlsListener := r.pandaCluster.SchemaRegistryAPITLS()
+	if tlsListener != nil && tlsListener.TLS.NodeSecretRef != nil {
```

Comment:
how come we're not using GetTLS method for these as that's going through the APILisytener interface?